### PR TITLE
GEOMESA-1269 Docs: Accumulo runtime required for ingest

### PR DESCRIPTION
**Pull request by mdzimmerman**
Moves runtime installation before command-line tools install.

Signed-off-by: Matthew Zimmerman <matt.zimmerman@ccri.com>

### CODE REVIEW

**Comment by jahhulbert-ccri** [comment]
@mdzimmerman can we add something about versioning somewhere? the client must match the distributed runtime and they need to be kept in sync.

**Comment by mdzimmerman** [comment]
The "Upgrading" section near the bottom of this chapter does mention matching versions: https://github.com/ccri/geomesa/blob/0eb95ce655b7d1d165192c6ac6f2f02e28ad2855/docs/user/installation_and_configuration.rst#upgrading

That having been said we could add a note box in this section about that.

I am not so sure we explicitly mention that data ingested from previous versions can be read with newer versions. Has that been true for all point releases in the 1.2.x series? I was under the impression (perhaps mistakenly) that this was not entirely the case for 1.2.2.

**Comment by jahhulbert-ccri** [comment]
@mdzimmerman yeah you can use data ingested from rc7 as well.

File: docs/user/jobs.rst [hunk 1]
```diff
@@ -130,7 +132,7 @@ slightly):
 
 .. code-block:: shell
 
-    yarn jar geomesa-jobs/target/geomesa-jobs-1.0.0-shaded.jar \
+    geomesa$ yarn jar geomesa-jobs/target/geomesa-jobs-1.0.0-shaded.jar \
```
Comment (jahhulbert-ccri) [inline docs/user/jobs.rst]: should this also have the $VERSION plugged in?

**Pull request merged**